### PR TITLE
Display server URL for Docker registry once again

### PR DESCRIPTION
**Pull request by a-roberts**
# Changes
For https://github.com/tektoncd/dashboard/issues/1107

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/dashboard/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: src/components/CreateSecret/Form/Annotations.js [hunk 1]
```diff
@@ -40,6 +43,13 @@ const Annotations = props => {
     defaultMessage: 'Docker Registry'
   });
 
+  function determineServerURLValue(annotation) {
+    console.log(annotation);
```
Comment (AlanGreene) [inline src/components/CreateSecret/Form/Annotations.js]: 🙈

File: src/components/CreateSecret/Form/Annotations.js [hunk 2]
```diff
@@ -40,6 +43,13 @@ const Annotations = props => {
...
+    if (annotation.access === 'git') {
+      return defaultGithubServerURL;
+    }
+    return defaultDockerServerURL;
+  }
```
Comment (AlanGreene) [inline src/components/CreateSecret/Form/Annotations.js]: with this change I can no longer edit the URL
we should only use the default value if `annotation.value` is not set / a default value (i.e. switching from git to docker)

Comment (a-roberts) [reply src/components/CreateSecret/Form/Annotations.js]: Thanks for taking a look, updated - have a look at the gotcha I've mentioned though please 👀

File: src/containers/CreateSecret/CreateSecret.js [hunk 3]
```diff
@@ -121,10 +124,33 @@ export /* istanbul ignore next */ class CreateSecret extends Component {
...
+      const previousAccessType = annotations[index].access;
+      // We enter this when it's the same access type, so a user is
+      // simply changing the value for server URL - set it to that
+      if (currentAccess === undefined) {
+        annotations[index][type] = value;
+      }
+      // Gotcha: it'd be useful to switch on access type (Git vs Docker) but
+      // actually that's passed in inconsistently, the strings to check on
+      // are translated ones so this assumes Docker and Git don't become something
+      // other than Docker and Git
```
Comment (AlanGreene) [inline src/containers/CreateSecret/CreateSecret.js]: this is fair, tool / product names generally aren't translated

**Pull request merged**